.babelrc

```diff
@@ -14,6 +14,9 @@
       "helpers": true,
       "polyfill": true,
       "regenerator": true
+    }],
+    ["transform-object-rest-spread", {
+      "useBuiltIns": true
```

Comment:
Will we need to add this (or should we add this) to the bpanel-cli plugin template?

Reply:
It gives a warning when running -mocha --compilers js:babel-core/register tests` that `--compilers` is going to be deprecated, so maybe if we first fix that